src/components/AirdropWalletsCarouselItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+import { AirdropWalletFooter } from './AirdropWalletFooter';
+
+const i18n = require('../../loc');
+
+const airdropGoals: AirdropGoal[] = [
+  { threshold: 5, name: i18n.airdrop.walletsCarousel.shrimp },
+  { threshold: 25, name: i18n.airdrop.walletsCarousel.crab },
+  { threshold: 100, name: i18n.airdrop.walletsCarousel.shark },
+  { threshold: 1000, name: i18n.airdrop.walletsCarousel.whale },
+];
+
+const readableOrder = [i18n.order.first, i18n.order.second, i18n.order.third, i18n.order.fourth];
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletsCarouselItem: FC<Props> = ({ walletDetails }) => {
+  const unreachedGoals = airdropGoals.filter((goal: AirdropGoal) => goal.threshold > walletDetails.balance);
+  // TODO: edge case of "all goals reached" not covered by designs. Awaiting UX input. For now returning last one - "whale"
+  const nextGoal = unreachedGoals.length > 0 ? unreachedGoals[0] : airdropGoals[airdropGoals.length - 1];
+  const nextGoalIndex = airdropGoals.findIndex((goal: AirdropGoal) => goal.threshold === nextGoal.threshold);
```

Comment:
hm... similarly as above, if I'm not wrong it can be expressed simpler:
```
const nextGoalIndex = airdropGoals.indexof(nextGoal)
```

Reply:
I believe it will not work, as object are not compered by fields value, but by references and reference will be different. Will check to be sure though

Reply:
Yup, not working like that. Reference to the object is different :) 
![image](https://user-images.githubusercontent.com/433674/104714740-e3ab6880-5725-11eb-9e39-496318586741.png)

Reply:
Okokok, good to know, thanks